exercises/practice/yacht/yacht.py

```diff
@@ -1,17 +1,3 @@
-"""
```

Comment:
Should the stub still contain a module docstring?

Reply:
No.  Since it referred to Python 2.x, and the equivalent in 3.x would be _either_ constants or `Enums` - and we aren't requiring `Enums`, it's best in this practice exercise to omit the docstring altogether.  See linked issue for discussion.